en/migrations.rst

```diff
@@ -204,14 +204,17 @@ Columns definition
 When using columns in the command line, it may be handy to remember that they
 follow the following pattern::
 
-    fieldName:fieldType[length]:indexType:indexName
+    fieldName:fieldType?[length]:indexType:indexName
 
 For instance, the following are all valid ways of specifying an email field:
 
+* ``description:string?``
```

Comment:
@dereuromark : I think I edited my comment and hit save after your reply... sorry about that